demo/src/app/components/accordion/accordion.component.html

```diff
@@ -1,8 +1,8 @@
-<ngbd-component-docs [name]="accordionDocs.name"
+<ngbd-component-docs [name]="accordionDocs.selector"
```

Comment:
We do have `className`. Oh, while we are at it -> we should probably also use `fileName` to:
* let people know how to import this thing
* have a link to sources on GH